### DISCUSSION TITLE
Mark the changes that are required to pass the review

### DISCUSSION THREAD

**Discussion by quilicicf** [community/community#120842] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

One thing I struggle with in a PR is marking the severity of the comments.

Basically, I like to annotate my comments with severity (not unlike what [the folks at Netlify do](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify/)).

The problem I have is that I need to have the pull request status reflect the comments:
* blocking issue => request changes
* major issue => no approval until it's discussed
* only minor issues => approve

And I need to have the author recognize what comment is at what level, potentially allowing them to sort by severity to address in descending order (and sometimes, minor issues asynchronously in another PR). 

I currently do this with labels starting my review messages and leveraging the search feature of the web browser but that's not very integrated and the link between comment severity and PR status is manual.

I'd really like as a reviewer to have the capability of marking the comments' levels and have the right PR status pre-selected when I submit my review.

I'd really like as an author to have the ability to get the list of comments by severity to address them in descending order (and understand at a glance why the PR status is what it is).

**Comment by Kolass2004** [comment]
It sounds like you need a more integrated solution for managing comment severity and PR status. One approach could be to use a code review tool or platform that supports custom tagging or labeling of comments, along with automated status updates based on those tags. Have you considered using tools like GitHub with custom labels or integrating with third-party code review tools that offer more flexibility in managing comment severity? This could streamline the process and make it easier for both reviewers and authors to track and address comments effectively.

**Reply by quilicicf** [reply]
I have never tried third party tools for code review, do you have recommendations? 
I still think that it would be something nice to add to GH's PRs though. The PR status today seems just incomplete to me as is, it'd be nice for GH to up their game here.
